include/oneapi/dpl/internal/random_impl/discard_block_engine.h

```diff
@@ -205,7 +205,7 @@ class discard_block_engine
     }
 
     _Engine engine_;
-    int n_ = 0;
+    size_t n_ = 0;
```

Comment:
It should be `::std::size_t` (actually, in every place of use) and it has missing include (e.g. `cstddef`). I am wondering how tests are passed. Seems like you don't include API headers first (before standard headers) or have some kind of umbrella header. Anyway, this header (and, probably, others) is not self-contained.

Reply:
"std" namespace for size_t and missing includes are added